code/game/objects/items/weapons/mop.dm

```diff
@@ -1,3 +1,5 @@
+#define is_cleanable(A) (istype(A, /obj/effect/decal/cleanable) || istype(A, /obj/effect/rune))

```

Comment:
>Removes overlays from cleaning

Intentional; there's not many overlays and none of them really need to be cleanable (AI hologram and holosigns, for example).